index.html

```diff
@@ -64,5 +64,159 @@ <h1 class="headline-alexandra_small">
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+
+      <div class="tonic">
+        <div><img src="images/proffesional.png" alt="" /></div>
+        <div class="right">
+          <h1>
+            Multi-Post<br />
+            Stories
+          </h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <h1>About Myself</h1>
+      <p class="description">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hesitate to
+        contact me.
+      </p>
+
+      <p class="connect">lets connect</p>
+      <div class="icons">
+        <a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        <a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        <a href=""><img src="images/Vector.png" alt="medium-icon" /></a>
+        <a href=""><i class="fa fa-github" aria-hidden="true"></i></a>
+        <a href=""><i class="fa fa-hand-peace-o" aria-hidden="true"></i></a>
+      </div>
+      <div class="resume">
+        <a href="" class="resume-link">Get my resume </a>
+      </div>
+      <div class="modals">
+        <div class="option">
+          <span>Languages</span>
+          <i class="fa-solid fa-arrow-down"></i>
+        </div>
+        <ul>
+          <li>
+            <div><img src="images/javascript.png" alt="" /></div>
+            <p>JavaScript</p>
+          </li>
+          <li>
+            <div><img src="images/html.png" alt="" /></div>
+            <p>HTML</p>
+          </li>
+          <li>
+            <div><img src="images/css.png" alt="" /></div>
+            <p>CSS</p>
+          </li>
+        </ul>
+        <div class="option">
+          <span>Frameworks</span>
+          <i class="fa-solid fa-arrow-right"></i>
+        </div>
+
+        <div class="option">
+          <span>Skills</span>
+          <i class="fa-solid fa-arrow-right"></i>
+        </div>
+      </div>
```

Comment:
- [ ] **Good job so far 👏 👏** However, It would be better to use a multilevel list instead of using separated lists. as a multilevel list can also improve the accessibility of your content by providing a clear and structured outline of the information. 🙏 

Example:
```
<ul class="modals">
  <li>
    <span>Languages</span>
    <i class="fa-solid fa-arrow-down"></i>
    <ul>
      <li>
        <img src="images/javascript.png" alt="" />
        <p>JavaScript</p>
      </li>
      <li>
        <img src="images/html.png" alt="" />
        <p>HTML</p>
      </li>
      <li>
        <img src="images/css.png" alt="" />
        <p>CSS</p>
      </li>
    </ul>
  </li>
  
  <li>
    <span>Frameworks</span>
    <i class="fa-solid fa-arrow-right"></i>
  </li>

  <li>
    <span>Skills</span>
    <i class="fa-solid fa-arrow-right"></i>
  </li>
</ul>